parsers/CA_QC.py

```diff
@@ -0,0 +1,131 @@
+from datetime import timedelta, timezone
+import logging
+
+# The arrow library is used to handle datetimes
+#import arrow
+
+# The request library is used to fetch content through HTTP
+import requests
+# pandas processes tabular data
+#import pandas as pd
+
+from pprint import pprint
+import json
+
+PRODUCTION_URL = "https://www.hydroquebec.com/data/documents-donnees/donnees-ouvertes/json/production.json"
+CONSUMPTION_URL = "https://www.hydroquebec.com/data/documents-donnees/donnees-ouvertes/json/demande.json"
+
+
```

Comment:
One more thing I forgot in the first pass: timezones 😄 

Here's how it could be done (inspired by `CA_ON` parser):

```py
# TODO: Add this after the constants
# Timezone object for local time of the parser 
tz_obj = timezone(timedelta(hours=-4), name="GMT-4")



# TODO: Use it like this for both production and consumption

"datetime": arrow.get(elem["date"], tzinfo=tz_obj).datetime,
```

Is the UTC offset correct and is DST something we should consider?

I have asked in https://github.com/tmrowco/electricitymap-contrib/discussions/3210#discussioncomment-1073430 whether the data is in local time or not :)